src/java.base/share/classes/module-info.java

```diff
@@ -22,6 +22,11 @@
  * or visit www.oracle.com if you need additional information or have any
  * questions.
  */
+/*
```

Comment:
Please leave a blank line before the copyright notice (like you added in `SunEntries.java`).

Reply:
Please do this for each IBM copyright notice.